lib/segment/src/common/utils.rs

```diff
@@ -766,6 +766,48 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_get_null_and_absent_values() {
+        let map = serde_json::from_str::<serde_json::Map<String, Value>>(
+            r#"
+            {
+                "a": null,
+                "b": [null, null],
+                "c": []
+            }
+            "#,
+        )
+        .unwrap();
+
+        assert_eq!(
+            get_value_from_json_map("a", &map).as_slice(),
+            &[&Value::Null],
+        );
+
+        assert!(get_value_from_json_map("a[]", &map).is_empty());
+
+        assert_eq!(
+            get_value_from_json_map("b", &map).as_slice(),
+            &[&Value::Array(vec![Value::Null, Value::Null])],
+        );
+
+        assert_eq!(
+            get_value_from_json_map("b[]", &map).as_slice(),
+            &[&Value::Null, &Value::Null],
+        );
+
+        assert_eq!(
```

Comment:
Isn't `c` by itself `is_empty()` as well now? 

Or is an empty array not considered empty?

Reply:
`get_value_from_json_map` should return `vec![c]`, i.e. a vector containing an empty array.

My PR doesn't change any behavior of `get_value_from_json_map`.

Reply:
Thank you for confirming.